julia_con/related.jl

```diff
@@ -1,13 +1,17 @@
+using Pkg
+Pkg.instantiate()
+
 using Base.Threads
 
 using JSON3
 using StructTypes
 using Dates
-using StaticArrays
 using ChunkSplitters
 
 # warmup is done by hyperfine
 
+include("../julia/vector5.jl")
```

Comment:
Is this permissible or should I duplicate the file?

Reply:
all good